base/options.go

```diff
@@ -44,6 +45,7 @@ func (o *Options) Parse(fs *flag.FlagSet, argv []string) error {
 	fs.StringVar(&o.ErrReportConv, "err-report-conv", os.Getenv("BOT_ERR_REPORT_CONV"),
 		"Conversation name or ID to report errors to")
 	fs.StringVar(&o.DSN, "dsn", os.Getenv("BOT_DSN"), "Bot database DSN")
+	fs.StringVar(&o.MultiDSN, "multi-dsn", os.Getenv("BOT_MULTI_DSN"), "Bot multi coordination database DSN")
```

Comment:
Where is the schema for this table? Why have a separate DSN?

Reply:
Forgot schema, I wanted this to act as kind of an addon, so I made is a separate DSN.

Reply:
I'm ok with an extra flag but it should just be soemting like `--enable-heartbeat` instead of new DSN imo

Reply:
@mmaxim what do you think about changing this to a bool flag instead?

Reply:
I think it is more flexible this way, if someone wants to just create the table in their own db they just pass the flag twice. What is the downside here, it's just a little harder to use?